### DISCUSSION TITLE
MD Energy printout

### DISCUSSION THREAD

**Discussion by daniel01-cpu** [grimme-lab/xtb#1067] [General]
Dear All,

I am exploring the MD features of XTB to calculate the vibrational spectrum using the autocorrelation function so the energy at every step is required. is it possible to dump the energy at every step but avoiding the geometry printout of the xtb.trj file?

Thanks!
Best
Daniel

**Comment by icamps** [comment]
A work around that I use is extracting the energies for each `.trj` file in a directory using the shell script below:

```
#!/bin/bash

for filename in *.trj; do
   energy=$(grep "energy: " $filename | awk '{printf "%12.9f \n",$2}')
cat -n > energy_$filename.dat <<!
$energy
!
done
```

**Comment by daniel01-cpu** [comment]
Thanks for the advice!
